app/src/main/java/org/woowatechcamp/githubapplication/presentation/home/notifications/adapter/NotiAdapter.kt

```diff
@@ -37,13 +38,10 @@ class NotiAdapter : ListAdapter<NotiModel, NotiAdapter.NotiViewHolder>(
         fun bind(item : NotiModel) {
             binding.noti = item
             binding.apply {
-                CoroutineScope(Dispatchers.IO).launch {
-                    val inputStream = URL(item.imgUrl).openStream()
-                    val bitmap = BitmapFactory.decodeStream(inputStream)
-                    withContext(Dispatchers.Main) {
-                        ivNoti.setImageBitmap(bitmap)
-                    }
-                }
+                Glide.with(binding.root)
+                    .load(item.imgUrl)
+                    .centerCrop()
+                    .into(binding.ivNoti)
```

Comment:
스코프 함수를 활용하셨는데, 그럴 경우에 아래처럼만 표현하셔도 괜찮을 것 같습니다!
```suggestion
                Glide.with(root)
                    .load(item.imgUrl)
                    .centerCrop()
                    .into(ivNoti)
```

Reply:
일단 bitmap 을 받아오는 걸 해본적이 없어서 익숙했던 glide 를 사용해봤습니다
그리고 compose 에도 coil 이 지원된다고 해서 coil 도 해볼게요 ㅎ